content/en/blog/2020/new-deployment-model/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: New Deployment Models for Istio
+subtitle: Clear Separation between Mesh Operator and Mesh Users
+description: A new deployment model for Istio.
+publishdate: 2020-08-19
+attribution: "Lin Sun (IBM), Iris Ding (IBM)"
+keywords: [multicluster,install,deploy,'1.7']
+---
+
+## Background
+
+In working with various service mesh users and vendors, we believe there are 3 key personas for a typical service mesh:
+* Mesh Operator
+* Platform Owner
+* Service Owner
+
+It is common to have all these personas work on the same clusters without any clear separation.  For example, there are multiple ways to deploy Istio according to the docs, which all started with mesh operator, platform owner and service owner sharing the single cluster first and then gradually expanding the mesh to multiple clusters or VMs.  None of these provided a clear separation between mesh operator and platform/service owner at the boundary of a cluster.  You may be thinking you could set up [Kubernetes RBAC rules](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) and namespaces to control which personas can do what within the cluster, however, sometimes you need stronger separation between the istio control plane and the rest at the cluster level.  Introduce a new deployment model for Istio which enables mesh operators to install and manage mesh control plane on dedicated clusters, separated from the data plane clusters.  This new deployment model could easily enable Istio vendors to run Istio control plane for service mesh users while users can focus on their workloads and Istio resources without worrying about installing or managing the Istio control plane.
+
+
+## New Deployment model
+
+Let’s take a close look of what has been deployed in your cluster if you install Istio following the default install:
+* Deployment and Services:
+    * Istiod: Istio control plane component, which includes an XDS server, CA server and a webhook server.
+    * Gateway: the default Istio ingress and egress gateways.
+* Configs:
+    * CRDs: Various Istio CRDs such as Gateway, Virtual Services, Destination Rules etc.
+    * ConfigMaps: Istio mesh configurations and Injection templates.
+    * Webhook configurations: configurations for two Istio webhooks (validation webhook and sidecar injector webhook)
+    * Service account, role binding, etc: Istio security settings
+    * Secrets: stores key and certs and credentials related to service account or Istiod.
+
+Below diagram shows how resources are deployed in the cluster after following the [default install](https://istio.io/latest/docs/setup/install/istioctl/#install-istio-using-the-default-profile).
+
+{{< image
+    link="./default-install.jpeg"
+    caption="default deployment model for istio"
+>}}
+
+With istio 1.7, it is possible to run Istiod on a separate cluster like the diagram below.
+
+{{< image
+    link="./central-istiod-single-cluster.jpeg"
```

Comment:
I think this diagram exposes too much lower level complexity for this blog post. I suggest making a new simpler diagram - I'm happy to draw one.

Reply:
Agreed, I'll rework based on the diagrams from deployment model.